pkg/asset/asset.go

```diff
@@ -210,9 +211,16 @@ func (as Assets) Get(name string) (Asset, error) {
 }
 
 func (as Assets) WriteFiles(path string) error {
-	if err := os.Mkdir(path, 0755); err != nil {
+	if err := os.MkdirAll(path, 0755); err != nil {
 		return err
 	}
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		return err
+	}
+	if len(files) > 0 {
+		return errors.New("Asset directory must be empty.")
```

Comment:
Asset -> asset. Error strings should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings